persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java

```diff
@@ -12,13 +12,17 @@
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
+import java.sql.ResultSet;
 import java.util.*;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration("/testContextDatabase.xml")
 @Configurable
 public class AlterationMyBatisRepositoryTest {
 
+    @Autowired
+    private org.springframework.jdbc.datasource.DriverManagerDataSource datasource;
```

Comment:
Do we need this?

Reply:
Fixed in [5d29809](https://github.com/cBioPortal/cbioportal/pull/10092/commits/5d298094d42bba02543f79a930e19ffc847b668a)